extensions/proposals/KHR_parallel_shader_compile/extension.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- vi:set sw=2 ts=4: -->
+<?xml-stylesheet href="../../extension.xsl" type="text/xsl"?>
+<proposal href="proposals/KHR_parallel_shader_compile/">
+  <name>KHR_parallel_shader_compile</name>
+
+  <contact> <a href="https://www.khronos.org/webgl/public-mailing-list/">WebGL
+  working group</a> (public_webgl 'at' khronos.org) </contact>
+
+  <contributors>
+    <contributor>Members of the WebGL working group</contributor>
+  </contributors>
+
+  <number>XYZ</number>
+
+  <depends>
+    <api version="1.0"/>
+  </depends>
+
+  <overview>
+    <mirrors href="https://www.khronos.org/registry/OpenGL/extensions/KHR/KHR_parallel_shader_compile.txt"
+             name="KHR_parallel_shader_compile">
+    </mirrors>
+
+    <features>
+      <feature>
+        Shader compilation and program linking may be performed in a separate CPU thread.
+      </feature>
+    </features>
+  </overview>
+
+  <idl xml:space="preserve">
+    [NoInterfaceObject]
+    interface KHR_parallel_shader_compile {
+      const GLenum MAX_SHADER_COMPILER_THREADS_KHR      = 0x91B0;
+      const GLenum COMPLETION_STATUS_KHR                = 0x91B1;
+	
+      void maxShaderCompilerThreadsKHR(GLuint count);
+    };
+  </idl>
+
+  <samplecode xml:space="preserve">
+    <pre>
+    var canvas = document.createElement("canvas");
+    var gl = canvas.getContext("webgl");
+    var ext = gl.getExtension('KHR_parallel_shader_compile');
+    // Don't do this unless you really want to override the implementation-specific maximum.
+    ext.maxShaderCompilerThreadsKHR(3);
```

Comment:
Let's update this code sample to show the best practices. Please:
 - Put the code which uses the extension object in "if (ext) { ... }" blocks, to make it clearer that this extension might not be supported
 - Query MAX_SHADER_COMPILER_THREADS_KHR
 - Set it to the maximum of (2, getParameter(MAX_SHADER_COMPILER_THREADS_KHR))

Reply:
The initial value of MAX_SHADER_COMPILER_THREADS_KHR is 0xFFFFFFFF according to the spec. If users didn't set it, the query returns 0xFFFFFFFF always.

Reply:
Done